paddle/fluid/eager/tensor_wrapper.h

```diff
@@ -94,15 +105,52 @@ class TensorWrapper {
       intermidiate_tensor_.set_autograd_meta(
           std::static_pointer_cast<paddle::experimental::AbstractAutogradMeta>(
               p_ab_autograd_meta));
+      check_inplace_version();
```

Comment:
Looks like we're gonna check inplace version anyway, let's move this function "check_inplace_version" out.

Reply:
done in PR https://github.com/PaddlePaddle/Paddle/pull/41118